src/multipart.jl

```diff
@@ -53,9 +53,8 @@ function Base.read(f::Form, n::Integer)
     return result
 end
 
-function Form(d)
+function Form(d; boundary=string(rand(UInt128), base=16))
```

Comment:
If we're allowing people to pass in a boundary, it might be helpful to assert they pass a string of the right length

Reply:
Good idea, I can open another PR in the morning.